canonical/lxd-ui
# Improve images

## Done

- address various issues around image handling discovered during in person review. Notes are in [this doc](https://docs.google.com/document/d/1BehzsvBhTVXrSrlSjFUoo7eMppzwmNLNfYQ_aDhfgi4/edit#heading=h.454p4yeusdrj)
- Add minimal image server
- Show image source in image selection modal
- Show local images in image selection modal
- update images from ubuntu and ubuntu-minimal servers

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @lorumic or @edlerd for access.
    - With a local copy of this branch, run as described [here](../HACKING.md#setting-up-for-development).
2. Perform the following QA steps:
    - create instance > ensure the image selection modal works as expected with 1. ubuntu source 2. ubuntu minimal source and 3. local image source
